src/apps/pages/RecordDetail/PlaceMap.tsx

```diff
@@ -0,0 +1,43 @@
+import { Peripleo, RuntimeConfig } from '@peripleo/peripleo';
+import { LocationMarkers } from '@performant-software/geospatial';
+import { Peripleo as PeripleoUtils } from '@performant-software/core-data';
+import Map from "@components/Map";
+import TranslationContext from '@contexts/TranslationContext';
+import { useTranslations } from '@i18n/client';
+
+interface Props {
+  record: any;
+  lang: string;
+}
+
+const PlaceMap = (props: Props) => {
+  if (!props.record.place_geometry?.geometry_json) {
+    return null;
+  }
+
+  const { t } = useTranslations();
+
+  return (
+    <TranslationContext.Provider
+      value={{ t, lang: props.lang }}
+    >
+      <RuntimeConfig
+        path='/config.json'
+        preprocess={PeripleoUtils.normalize}
+      >
+        <Peripleo>
+          <Map>
+            <LocationMarkers
+              boundingBoxOptions={{
+                animate: false
+              }}
+              data={props.record.place_geometry.geometry_json}
```

Comment:
This should probably have some `null` checking. Is it possible to get to this page for a place without a geometry?

Reply:
There's a null check before the return.